src/Eval.hs

```diff
@@ -483,9 +483,7 @@ catcher ctx exception =
 
 specialCommandWith :: Context -> XObj -> SymPath -> [XObj] -> IO (Context, Either EvalError XObj)
 specialCommandWith ctx xobj path forms = do
-  let pathStrings = contextPath ctx
```

Comment:
pathStrings use here is probably also a potential bug/cleanup--I think for almost all if not all of these lookups we should not be using `path` but rather:

`consPath (union (contextPath context) modulePrefixes) path`

where `modulePrefixes` are the list of prefixes in the given `SymPath` argument--this ensures we lookup the right things when we're evaluating something in some local context. I could be wrong about this, though @hellerve @eriksvedang

Reply:
Hmm I'm not sure without thinking more about it. Maybe create an issue for it? @scolsen